puppet/modules/slave/manifests/init.pp

```diff
@@ -251,6 +251,12 @@
     package { 'google-chrome-stable':
       ensure => latest,
     }
+
+    include ::chromedriver
```

Comment:
chromedriver is also in EPEL. Currently at version 61.0.3163.100. Would that be new enough?